src/scss/components/ec-card/_ec-card.scss

```diff
@@ -4,7 +4,35 @@
 
   padding: 8px;
   background: $white;
-  overflow: hidden;
+  word-break: break-word;
+
+  &--is-interactive {
+    position: relative;
+    cursor: pointer;
+    transition: all 0.4s ease;
+
+    &:after {
+      @include box-shadow-level-2;
+
+      content: '';
+      position: absolute;
+      z-index: -1;
```

Comment:
Why this is needed, and we have a file where we define all the z-index, if this one is needed you can create one and used it here

Reply:
@jaimeperezpedra This is following the link shared by Milos, putting a pseudo-element behind and playing with the opacity having a better performance.